### PR TITLE
feat: Beta NFT Access

### PR DESCRIPTION
**Pull request by aashutoshrathi**

### CODE REVIEW

File: docs/components/Web3Provider.tsx [hunk 1]
```diff
@@ -7,15 +7,26 @@ import "@rainbow-me/rainbowkit/styles.css";
...
 import { WagmiProvider } from "wagmi";
 import { sepolia } from "wagmi/chains";
+import { walletConnectProjectId } from "../constants";
+
+// This is faster than using the default RPC
+const sepoliaEuRPC = {
+  ...sepolia,
+  rpcUrls: {
+    default: {
+      http: ["https://rpc2.sepolia.org/"],
```
Comment (kautukkundan) [inline docs/components/Web3Provider.tsx:17]: public RPC endpoint

File: docs/components/NFTMinted.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        .then((data) => setNftMeta(data));
+    }
+  }, [nftMetaUrl]);
+
+  return (
+    <div>
+      <span>Congrats, you have access to our SDK and own our NFT. 🥳</span>
+      <br />
+      <span>
+        Reach out to use on our{" "}
```
Comment (kautukkundan) [inline docs/components/NFTMinted.tsx]: #typo use -> us

File: docs/components/NFTMinted.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        <a
+          className="underline"
+          href={"https://discord.com/invite/PY4nhg7bcc"}
+          target="_blank"
+          rel="noreferrer"
+        >
+          Discord Channel
+        </a>{" "}
+        to get started with deploying
```
Comment (kautukkundan) [inline docs/components/NFTMinted.tsx]: "deploying" -> "building micro-rollups"

File: docs/components/NFTMinted.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+      </span>
+      <br />
+      <br />
+
+      <div className="flex justify-center">
+        {nftMeta ? (
+          <div className="flex-col text-center items-center">
+            <img src={nftMeta.image} alt={nftMeta.name} />
```
Comment (kautukkundan) [inline docs/components/NFTMinted.tsx:53]: will this support `mp4` or `gif`

**Review by kautukkundan — CHANGES_REQUESTED** [review]
Looks good
Some changes requested and Questions

File: docs/components/Web3Provider.tsx (hunk 1 above)
Comment (aashutoshrathi) [reply docs/components/Web3Provider.tsx:17]: Yup it public

File: docs/components/NFTMinted.tsx (hunk 4 above)
Comment (aashutoshrathi) [reply docs/components/NFTMinted.tsx:53]: GIF yes! Do we need MP4?

**Review by kautukkundan — APPROVED** [review]

**Pull request merged**